include/IECoreMaya/MessageHandler.h

```diff
@@ -55,7 +55,7 @@ class IECOREMAYA_API MessageHandler : public IECore::MessageHandler
 
 	private :
 
-		typedef tbb::mutex MsgMutex;
+		typedef std::mutex MsgMutex;
```

Comment:
I finally had a chance to check this at IE and found one small issue. This requires a matching change in the cpp

```
---------------------- src/IECoreMaya/MessageHandler.cpp ----------------------
index 7e72a0699..326106949 100644
@@ -42,7 +42,7 @@ void MessageHandler::handle( Level level, const std::string &context, const std:
 {
 
 	// MGlobal.display* seemingly is not thread safe.
-	MsgMutex::scoped_lock lock( m_mutex );
+	std::lock_guard<MsgMutex> lock( m_mutex );
 ```

Reply:
Thanks for catching that - force pushed a fix into d6391b6. I had intended to touch only files that I could compile and test here, but it seems I got a bit carried away with my search and replace.